aten/src/ATen/test/pow_test.cpp

```diff
@@ -110,39 +110,53 @@ void assert_eq(T val, T act, T exp) {
   ASSERT_EQ(act, exp);
```

Comment:
Are the tests in this file actually run in CI? We couldn't find the test output in several builds, including the ASAN build. If it's not run, should we continue to test this in C++ or port the tests to Python?

Reply:
(One reason to move this test to Python, for example, would be to run on CPU and CUDA easily.)

Reply:
most of the tests are in python. I had to update this one to get things to build, but I also generally prefer the python tests.